trie/codec_test.go

```diff
@@ -0,0 +1,48 @@
+package trie
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestKeyEncodeByte(t *testing.T) {
+	tests := []struct {
+		input    byte
+		expected byte
+	}{
+		{byte(36), byte(66)},
```

Comment:
could write as `{byte(0x24), byte(0x42)}` for more clarity, similarly 0xf0 and 0x0f below. not a huge deal